### DISCUSSION TITLE
Variante d'acteur boite de saisie

### DISCUSSION THREAD

**Discussion by aurelielefrWit** [witsa/synapps#604] [Ideas]
Bonjour, 

Je pense qu'il serait agréable d'avoir une boite de saisie avec une icone/image, à l'intérieur de la boite, comme dans l'interface de Synapps. Voir si dessous.
![image](https://github.com/witsa/synapps/assets/138686045/66cc7174-26b5-4549-9403-99e4327f83ef)
Mais aussi de choisir l'emplacement de l'icone.

Merci,😊

**Comment by manufitoussiwit** [comment]
Est ce quelqu'un se sent de faire un composite qui fait ça ? ;-)

**Reply by Raphwit** [reply]
Peut être bien :)
